src/languageservice/services/yamlCodeActions.ts

```diff
@@ -312,9 +316,52 @@ export class YamlCodeActions {
     }
     return results;
   }
+
+  /**
+   * Check if diagnostic contains info for quick fix
+   * Supports Enum/Const/Property mismatch
+   */
+  private getPossibleQuickFixValues(diagnostic: Diagnostic): string[] | undefined {
+    if (typeof diagnostic.data !== 'object') {
+      return;
+    }
+    if (
+      diagnostic.code === ErrorCode.EnumValueMismatch &&
+      'values' in diagnostic.data &&
+      Array.isArray((diagnostic.data as YamlDiagnosticData).values)
+    ) {
+      return (diagnostic.data as YamlDiagnosticData).values;
+    } else if (
```

Comment:
Is this else really needed?

Reply:
Do you mean quickFix for properties? or can you explain your point further?
 - I think that property quickFix can be useful
 - note that there are different property names in `diagnostic.data` object (`values` vs `properties`)